server/src/main/scala/viper/gobraserver/GobraServer.scala

```diff
@@ -51,50 +53,61 @@ object GobraServer extends GobraFrontend {
     _server.start()
     VerifierState.flushCachedDiagnostics()
   }
-  
 
-  private def serverExceptionHandling(fileData: FileData, resultFuture: Future[VerifierResult]) {
+  def restart() {
```

Comment:
We usually provide an explicit return type, even for Unit functions, i.e. `def restart(): Unit = { ... }`